readme.md

```diff
@@ -51,21 +51,6 @@ The `readEncoding` and `writeEncoding` can be any one of those accepted by
 
 `DuplexStream` is a [Duplex Stream][].
 
-#### Event: 'open'
```

Comment:
I didn't think that this event made sense since the file will be opened twice (once for reading, once for writing) so which file is the opened file?